style.css

```diff
@@ -5,189 +5,205 @@
-  width: 311px;
+  background-repeat: no-repeat;
+  background-position: 90%;
+  width:50%;
   height: 215px;
 }
-.menubar {
+.menu__bar{
   display: flex;
-  height: 32px;
-  justify-content: center;
-  align-items: end;
-  position: absolute;
-  top: 215px;
-}
-.menubar ul {
-  list-style-type: none;
 }
-.menubar li {
-  float: left;
+.menu__items{
+  width:50%;
 }
 
-.btndivider {
-  vertical-align: middle;
-  background-image: url(images/menu-devider.gif);
-  background-repeat: no-repeat;
-  display: inline-block;
-  width: 2px;
-  height: 32px;
-}
-.hidden {
-  background-image: none;
-}
-.menubar a {
+.menu__items ul{
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width:360px;
+  height:32px;
+  white-space: nowrap;
+}
+.menu__items li{
+  height:32px;
+}
+.menu__items a{
+  display:inline-block;
+  line-height:32px;
+  text-align: center;
+  width:100%;
   font-family: "Trebuchet MS", sans-serif;
   font-size: 10pt;
   font-weight: bold;
   text-decoration: none;
   color: white;
-  margin: 0 15px 0 15px;
 }
-.menubar li:first-child a:first-of-type {
-  margin: 0 15px 0 8px;
+
+.menu__divider {
+  background-image: url(images/menu-devider.gif);
+  background-repeat: no-repeat;
+  display: inline-block;
+  width: 2px;
+  height: 32px;
 }
-.menubar li:hover {
+
+.menu__bar a:hover {
   background-image: url(images/menu-hover.gif);
   background-repeat: no-repeat;
   background-position: top;
 }
-.mainbody {
-  width: 100%;
-  height: 800px;
-  background-color: rgb(239, 239, 239);
-  position: relative;
+.main__body {
+  display: flex;
+  justify-content: center;
+  height:700px;
 }
-.bodycontent {
+.body__content {
   display: flex;
-  position: absolute;
-  width: 744px;
-  height: 800px;
-  left: 50%;
-  top: -35px;
-  transform: translateX(-50%);
+  width: 720px;
 }
 .column {
-  width: 317px;
-  height: 800px;
-  padding: 20px 0 0 45px;
+  width: 50%;
+  height:700px;
 }
 .column.right {
+  position:relative;
+  top:-33px;
+  display:flex;
   background-color: white;
+  height:733px;
+  border-radius: 0 0 10px 10px;
 }
-.article-top {
-  margin-right: 20px;
+.column__content__container{
+  display: flex;
+  flex-direction:column;
+  justify-content: space-evenly;
+  max-height:100%;
+  margin:0 41px 0;
 }
-.articletitle {
+
+.article__heading {
   display: flex;
   width: 100%;
   align-items: center;
   margin-bottom: 20px;
 }
-.titleicon {
+.title__icon {
   width: 60px;
   height: 71px;
   flex-shrink: 0;
 }
-#updatesarticletitleicon {
+#updates__article__title__icon {
   background-image: url(images/image1.jpg);
   background-repeat: no-repeat;
 }
-.articleheading {
+.article__title {
   display: flex;
   flex-direction: column;
   width: 100%;
   margin-left: 10px;
 }
-.heading1 {
+.article__title .title {
   font-family: "Trebuchet MS", sans-serif;
   font-size: 20pt;
   font-weight: bold;
   color: #4d4d4d;
 }
-.heading2 {
+.article__title .subtitle {
   font-family: Arial, Helvetica, sans-serif;
```

Comment:
Cand, foarte multe elemente pe site folosesc aceleasi stiluri sau similare, e bine sa le setezi pe astea la nivel global dupa care in pagina sau sectiuni unde vrei anumite variatii ale stilurilor respective, vei putea sa aplici doar ce trebuie schimbat: 
 Gen 
```
body {
   p {
      font: blah;
      font-size: blah
   }
   h1 {
      font: blah;
   }

}
```

si dupa in sectiuni va trebui doar sa specifici stiluri pentru elementele care le vrei schimbate putin fata de default. Astfel nu va ma trebui sa te repeti si vei scrie mult mai putin cod ca sa obtii aceleasi rezultate.